eclipse-scout-core/src/desktop/hybrid/DisposeWidgetsHybridActionDo.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import {DoEntity} from '../..';
+
+export interface DisposeWidgetsHybridActionDo extends DoEntity {
```

Comment:
DOs for hybridAction need to have the _type property set, so maybe we should add it here.

Reply:
This is not necessary at the moment, it will be added anyway when DOs will be migrated to classes